src/components/VolumeControl.js

```diff
@@ -46,22 +46,31 @@ const VolumeControl = ({initialVolume, plusJoin, minusJoin, isMuted, volumeJoin}
     window.CrComLib.publishEvent('b', `${minusJoin}`, false);
   }
   const handleDecreaseOnMouseDown = () => {
+    clearInterval(pressInterval);
```

Comment:
Good approach. Note: it is often tricky to cover all the nuance of touchstart/mousedown events -- in this case, the touchend/mousebuttonup events are bound to the button, which is the expected behavior. A corner case, is that a user touchstarts on the button, slides their finger (by accident, etc), and releases it outside of the button bounding rect. In that case, your touchend might never fire (!). In this case, it might be helpful to capture touchends/mouseups on the `body` element, and if there is a button being pressed to trigger the callback for release on the pressed object.